.github/workflows/wheels.yml

```diff
@@ -154,7 +161,8 @@ jobs:
 
       - name: Build and test wheels
         env:
-          CIBW_PRERELEASE_PYTHONS: ${{ matrix.prerelease }}
```

Comment:
What happened to the `prerelease` variable? Was it not used anywhere else?

It looked like this PR was renaming it from `matrix.prerelease` to `matrix.prerelease_pythons` which made me expect to see some other renames as well.

Reply:
I think `matrix.prerelease` was used at one point for Python 3.12 before Python 3.12 was released and then once Python 3.12 was released, the line was removed